operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements.md

```diff
@@ -30,20 +31,30 @@ The Release Manager \(Amy Blais\) owns the release announcement, including the f
 
 1. **Exciting title**
    1. Do not lead with a competitor parity feature 
+   2. Communicate primary benefit
+   3. Lead with the most exciting/impactful new feature
+   4. Include 1-2 features
+     - Have clarity on why those 1-2 features were chosen for the title. Ask PMs to share specs or other details on why the features we are shipping were built
```

Comment:
1 - Should this be included as part of the blog post process above? Perhaps as a separate bullet before `Writer drafts announce with feedback` when you're working with PMs on the outline?
2 - Don't we already do this, at least partially, by writing down the benefits?
3 - Formatting seems to break for this bullet